tests/system/testdata/sessions/stop.txtar

```diff
@@ -26,6 +26,22 @@ ak sessions log ses_00000000000000000000000004 --skip -1 --no-timestamps -j
 return code == 0
 output equals_json file last.json
 
+ak deploy --manifest test.yaml --file test.star
+return code == 0
+
+ak sessions start --deployment-id dep_0000000000000000000000000a --entrypoint test.star:test
+return code == 0
+
+ak sessions watch ses_0000000000000000000000000b --no-timestamps --end-state RUNNING --timeout 7s
+return code == 0
+
+ak sessions stop ses_0000000000000000000000000b --reason test
+return code == 0
+
+ak sessions log ses_0000000000000000000000000b --no-timestamps -j
+return code == 0
+output equals file test.txt
```

Comment:
Why not use the regular `diff` command?

Reply:
When I run this manually from the command line the output of the log is different:
```
{"state":{"created":{}}}
{"state":{"running":{"run_id":"run_2sv8bkxxrf27qbeqkc4g4bancx"}}}
{"state":{"running":{"run_id":"run_2sv8bkxxrf27qbeqkc4g4bancx","call":{"function":{"executor_id":"run_2sv8bkxxrf27qbeqkc4g4bancx","name":"test","desc":{},"data":"dGVzdCMx"}}}}}
{"print":{"text":"started"}}
{"call_spec":{"function":{"function":{"executor_id":"int_00000000008c093f7e9fccbf69","name":"freeze","desc":{"input":[{"name":"duration","optional":true},{"name":"allow_cancel","optional":true}]}}},"args":[{"integer":{"v":"10"}},{"boolean":{"v":true}}],"seq":1}}
{"call_attempt_start":{}}
{"call_attempt_complete":{"retry_interval":"0s","is_last":true,"result":{"value":{"nothing":{}}}}}
{"print":{"text":"finished"}}
{"state":{"completed":{"prints":["started","finished"],"exports":{"test":{"function":{"executor_id":"run_2sv8bkxxrf27qbeqkc4g4bancx","name":"test","desc":{},"data":"dGVzdCMx"}}},"return_value":{"nothing":{}}}}}
{"stop_request":{"reason":"test"}}
```

Reply:
@daabr ? essentially what this does is a diff. i think when running yourself you're not running in test mode.